bee-queue/arena
# Add tree view when creating a flow and add ability for searching

#### Changes Made
This PR adds a tree view to show the flow when it is created and searching
#### Potential Risks
No potential risk
#### Test Plan
1. Go to example directory
2. Run npm run start:bullmq_with_flows
3. Go to first connection
4. Create a new flow, for example with this body:
```
{
    "name": "parent-job",
    "queueName": "name_of_my_parent_queue",
    "data": {},
"children": [
      {"name": "child", "data": {"idx": 2}, "queueName": "name_of_my_queue"}
    ]
}

```
5. Expect that a tree view is showed.
6. Also there is a new option for Search Flow.
7. You should pass 4 values, queueName, jobId, depht of the flow tree and limit Children, these params are passed to getFlow function to get a flow and be showed in the tree view
#### Checklist
